### PR TITLE
[google_maps_flutter_platform_interface] added getpoint to support projection method pointForMeters in iOS.

### PR DESCRIPTION
**Pull request by klaes-ashford**
## Description

This would enable [[google_maps_flutter] Add pointForMeters to plugin in iOS #2503
](https://github.com/flutter/plugins/pull/2503)

## Related Issues

https://github.com/flutter/plugins/pull/2503
https://github.com/flutter/plugins/pull/2503#issuecomment-620851802

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by aswinkviswanath** [comment]
@ditman I am unable to add reviewers for this PR. please review this. thanks.

File: packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart [hunk 1]
```diff
@@ -437,6 +437,13 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
     return channel(mapId).invokeMethod<double>('map#getZoomLevel');
   }
 
+  /// Converts a distance in meters to content size.
+  Future<double> getPoint(double meters, LatLng latLng, {@required int mapId}) {
+    assert(meters != null && latLng != null);
+    return channel(mapId).invokeMethod<double>('map#getPoint',
+        <String, dynamic>{"meter": meters, "location": latLng.toJson()});
+  }
```
Comment (ditman) [inline packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart:445]: `getPoint` sounds like a too generic method name for what this does. Can we call this `getPointsForMeters` so it follows the [native API](https://developers.google.com/maps/documentation/ios-sdk/reference/interface_g_m_s_projection#a7275229ff43ac7779c28979855c3c02f) more closely?

Also, the method name over the method channel can be: `projection#getPointsForMeters`? This'd leave the door open to creating a more complete set of `projection` calls.

I'm not an expert in these APIs, so I'm going to defer the final opinion to @cyanglaz!

Comment (aswinkviswanath) [reply packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart:445]: yes, this makes sense. will implement the changes once @cyanglaz confirms.

Comment (aswinkviswanath) [reply packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart]: this change has been implemented.

**Comment by aswinkviswanath** [comment]
review suggestions has been implemented. method channel has been renamed to `projection#getPointsForMeters` so that it'd leave the door open to creating a more complete set of projection calls as per @ditman's suggestion. @cyanglaz, please verify if this is the way to go.

File: packages/google_maps_flutter/google_maps_flutter_platform_interface/pubspec.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ description: A common platform interface for the google_maps_flutter plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.1
+version: 1.0.2
```
Comment (cyanglaz) [inline packages/google_maps_flutter/google_maps_flutter_platform_interface/pubspec.yaml:6]: we are adding a new public API, need to bump the minor version.
So 1.1.0

**Comment by klaes-ashford** [comment]
@googlebot I signed it!

**Comment by cyanglaz** [comment]
@klaes-ashford I left some comments on the PR for plugin. We will land this when the other PR is also ready.

**Pull request closed without merging**